### DISCUSSION TITLE
Foliate won't open library - solved

### DISCUSSION THREAD

**Discussion by larntz** [johnfactotum/foliate#990] [General]
I was having an issue where I could no longer use Foliate.

Every time I opened the application it would open the previous book. Unfortunately, when I wanted to go back to my library it would go to `Catalogs`, spin, and then crash. 

Found that to fix this I had to use this gsettings command:

```
gsettings set com.github.johnfactotum.Foliate.library page 'library'
```

It was previous set to: 
```
com.github.johnfactotum.Foliate.library page 'catalog'
```

I nearly gave up on using Foliate because of this. No idea what caused it but I will definitely be staying away from the Catalogs tab in the future. 

This was on Ubuntu 22.04.1 LTS.

**Comment by vijay-prema** [comment]
I had an issue where I could open local files but the catalogs never showed anything.  I'm using the Flatpak linux version. I just deleted the entire `~/.var/app/com.github.johnfactotum.Foliate/` folder which removes all of my settings, history and progress, and after that everything worked.
